core/src/main/java/com/linecorp/armeria/internal/common/DefaultJacksonObjectMapperProvider.java

```diff
@@ -42,6 +43,10 @@ public ObjectMapper newObjectMapper() {
         // Create the default ObjectMapper with the modules provided by SPI.
         jsonMapperBuilder.findAndAddModules();
         final ObjectMapper mapper = jsonMapperBuilder.build();
+
+        // Serialize java.time objects in standard ISO-8601 string representation.
+        mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
```

Comment:
Question) Could this be a breaking change for users?   I wasn't able to find a reason for this change in the issue or PR description. Can you describe the reasoning for this change? 😅

Reply:
If this option is enabled, the time ojects are serialized to numbers which are not human-readable. I thought it could be more sensible defaults. hanged the default option. Most JSON libraies may handles the ISO standard formats.
When I have to serialize java.time objects I always prefer ISO format styles over just numbers.

If you concerned the breaking changes, we may add a flag or remove the custom option and follow the default option of Jackson.

Reply:
I see. I think I'm fine with the change if most implementations accept both formats by default 👍

Reply:
Had a chat internally and I decided to revert the custom configuration because other frameworks such as Spring provide an `ObjectMapper` with the default options.